### PR TITLE
Add stories for chat info panel and notifications

### PR DESCRIPTION
**Pull request by akalin-keybase**
Fix some minor typing issues along the way.

### CODE REVIEW

**Comment by keybase-ci-visdiff** [comment]
:no_entry: Error rendering the commits 5cab7e43c582...e5213d5ae6f5 on linux. <details><summary>:mag_right: 4 errored, 6 changed</summary>

 * Error in 5cab7e43c582: **Chat_Info_Panel-Normal-ERROR**  
   Chat_Info_Panel-Normal-ERROR rendered
 * Error in e5213d5ae6f5: **Chat_Info_Panel-Normal-ERROR**  
   Chat_Info_Panel-Normal-ERROR rendered
 * Error in 5cab7e43c582: **Chat_Info_Panel-Muted-ERROR**  
   Chat_Info_Panel-Muted-ERROR rendered
 * Error in e5213d5ae6f5: **Chat_Info_Panel-Muted-ERROR**  
   Chat_Info_Panel-Muted-ERROR rendered
 * Changed: **Chat_Inbox-You_Rekey_Selected** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1504234020013-5cab7e43c582-e5213d5ae6f5-linux/Chat_Inbox-You_Rekey_Selected-5cab7e43c582.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1504234020013-5cab7e43c582-e5213d5ae6f5-linux/Chat_Inbox-You_Rekey_Selected-e5213d5ae6f5.png)  
   Chat_Inbox-You_Rekey_Selected comparison
 * Changed: **Chat_Inbox-You_Rekey** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1504234020013-5cab7e43c582-e5213d5ae6f5-linux/Chat_Inbox-You_Rekey-5cab7e43c582.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1504234020013-5cab7e43c582-e5213d5ae6f5-linux/Chat_Inbox-You_Rekey-e5213d5ae6f5.png)  
   Chat_Inbox-You_Rekey comparison
 * Changed: **Chat_Inbox-Part_Rekey_Selected** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1504234020013-5cab7e43c582-e5213d5ae6f5-linux/Chat_Inbox-Part_Rekey_Selected-5cab7e43c582.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1504234020013-5cab7e43c582-e5213d5ae6f5-linux/Chat_Inbox-Part_Rekey_Selected-e5213d5ae6f5.png)  
   Chat_Inbox-Part_Rekey_Selected comparison
 * Changed: **Chat_Inbox-Part_Rekey** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1504234020013-5cab7e43c582-e5213d5ae6f5-linux/Chat_Inbox-Part_Rekey-5cab7e43c582.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1504234020013-5cab7e43c582-e5213d5ae6f5-linux/Chat_Inbox-Part_Rekey-e5213d5ae6f5.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1504234020013-5cab7e43c582-e5213d5ae6f5-linux/Chat_Inbox-Part_Rekey.png)
 * Changed: **Chat_Inbox-Long_Top** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1504234020013-5cab7e43c582-e5213d5ae6f5-linux/Chat_Inbox-Long_Top-5cab7e43c582.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1504234020013-5cab7e43c582-e5213d5ae6f5-linux/Chat_Inbox-Long_Top-e5213d5ae6f5.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1504234020013-5cab7e43c582-e5213d5ae6f5-linux/Chat_Inbox-Long_Top.png)
 * Changed: **Chat_Banner_Broken-Broken_Tracker_3** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1504234020013-5cab7e43c582-e5213d5ae6f5-linux/Chat_Banner_Broken-Broken_Tracker_3-5cab7e43c582.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1504234020013-5cab7e43c582-e5213d5ae6f5-linux/Chat_Banner_Broken-Broken_Tracker_3-e5213d5ae6f5.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1504234020013-5cab7e43c582-e5213d5ae6f5-linux/Chat_Banner_Broken-Broken_Tracker_3.png)
</summary>

File: shared/chat/conversation/info-panel/index.js.flow [hunk 1]
```diff
@@ -1,6 +1,14 @@
 // @flow
 import {Component} from 'react'
 
+export type MuteRowProps = {
```
Comment (chrisnojima) [inline shared/chat/conversation/info-panel/index.js.flow]: if this is going to be a shared component (aka a plain .js file) we don't need a sep. flow file and its likely more dangerous to do so as they can drift out of sync

Comment (akalin-keybase) [reply shared/chat/conversation/info-panel/index.js.flow]: Done

**Comment by akalin-keybase** [comment]
PTAL

**Pull request merged**